### ISSUE TITLE
Error while executing request

### ISSUE THREAD

**Issue by FLab-Projects** [Tasshack/dreame-vacuum#751]
**Describe the bug**
An error occurs periodically: Error while executing request

I have a vacuum cleaner: xiaomi.vacuum.c102gl 
Configured via ru domain, work only via cloud services

**To Reproduce**
Just use the integration :)

**Expected behavior**
I would really like to have the ability to have local control so that I can take full control of the vacuum cleaner. I’m far from a developer, so I don’t know what the source of the problem is: Xiaomi servers or errors in integration

**Additional Information (please complete the following information)**
 - Model Name [e.g. dreame.vacuum.p2028]: xiaomi.vacuum.c102gl
 - Firmware Version [e.g. 1156]: 1080
 - Home Assistant Version: 2024.10.1
 - Configuration Type [With or without map support]: with map support, cloud only
 - Errors or warnings shown in the HA logs (if applicable):

```

Эта ошибка возникла в пользовательской интеграции.

Регистратор: custom_components.dreame_vacuum.dreame.protocol
Источник: custom_components/dreame_vacuum/dreame/protocol.py:1006
интеграция: Dreame Vacuum (документация, проблемы)
Первое сообщение: 04:22:24 (83 сообщений)
Последнее сообщение: 21:12:52

Error while executing request: https://ru.api.io.mi.com/app/v2/home/rpc/1065439715 HTTPSConnectionPool(host='ru.api.io.mi.com', port=443): Read timed out. (read timeout=5)
Error while executing request: https://ru.api.io.mi.com/app/user/get_user_device_data HTTPSConnectionPool(host='ru.api.io.mi.com', port=443): Read timed out. (read timeout=5)

```

In conclusion, I would like to thank you from the bottom of my heart for the wonderful integration. I am ready to help in debugging and solving the problem in any way I can. Thank you for your work❤️
